src/index.js

```diff
@@ -45,14 +42,19 @@ let ElasticsearchProvider = (config = { request: {} }) => ({
             ...(scroll && { sort: ['_doc'] }),
             ...aggs,
           },
-        })
+        }
 
     let client = config.getClient()
+    let cached = redisCache(config, schema)
```

Comment:
if we can, we might want to move this inline so that we're not doing any extra set up work if we aren't going to use the cache (e.g. for scan/scroll in exports)